ibm/mas_devops/roles/suite_db2_setup_for_facilities/templates/db2/scripts/db2configdb.sh.j2

```diff
@@ -0,0 +1,268 @@
+#!/bin/sh
+# 
+# Licensed Materials - Property of IBM
+# 
+# Restricted Materials of IBM
+# 
+# (C) COPYRIGHT IBM CORP. 2014.  All Rights Reserved.
+# 
+# create DB2 database for use by TRIRIGA
+# set database level performance and other settings
+# 
+# requirements -
+# database name must not be used on the server
+# instance must be valid and started
+# territory must be valid with UTF-8 code set
+# must be run as DB2 instance user
+# tririga user must be a valid user on the system
+
+# db2createdb name instance territory installDir tririgaUser
+
+# echo "Validating request..." 
+# if [ "$#" -ne "6" ]; then 
+#    echo "Invalid usage - db2createdb name instance territory installDir tririgaUser"
+#	echo "name - database to be created"
+#	echo "instance - instance to create the database for"
+#	echo "territory - territory, must be consistent with UTF-8"
+#	echo "installDir - DB2 installation directory"
+#	echo "tririgaUser - tririga user name - user must exist"
+#	echo "schema - tririga database schema"
+#	exit 1
+# fi
+
+# set parameters used in the script
+
+DB2_DBNAME={{ db_name }}
+DB2_INSTNAME='db2inst1'
+DB2_TERRITORY='US'
+DB2_HOME='/mnt/blumeta0/home/db2inst1/sqllib'
+DB2_USER={{ db_username }}
+DB2_SCHEMA={{ db_schema }}
+
+# validate running as instance user
+
+if [[ `whoami` != "$DB2_INSTNAME" ]]; then
+	echo "Error: this script needs to be run as the instance user, $DB2_INSTNAME"
+	exit 1
+fi
+
+# start the instance
+
+echo "Starting the instance, $DB2_INSTNAME if it is not already started..."
+echo "db2start"
+db2start
+rc=$?
+if [ $rc -ne "0" ]; then
+	if [ $rc -ne "1" ]; then
+    	echo "Error unable to start instance $DB2_INSTNAME for some reason = $rc.  Cannot continue and create the database."
+    	exit 1
+	fi
+fi
+echo "Return code for starting instance $DB2_INSTNAME is $rc"
+
+ok=0
+
+# Create the database - pagesize of 32 K required, UTF-8 required if any MBCS data will be used
+
+echo "Processing batch command file has started please wait..."
+#echo "$DB2_HOME/bin/db2 create db $DB2_DBNAME ALIAS $DB2_DBNAME using codeset UTF-8 territory $DB2_TERRITORY pagesize 32 K"
+#$DB2_HOME/bin/db2 create db $DB2_DBNAME ALIAS $DB2_DBNAME using codeset UTF-8 territory $DB2_TERRITORY pagesize 32 K
+#rc=$?
+#if [ $rc -ne "0" ]; then
+#	echo "Unable to create database for some reason = $rc"
+#	echo "Possible errors include invalid parameters passed into the script, database name in use, or not running as the $DB2_INSTNAME user"
+#	exit $rc
+#else
+#	echo "Return code for creating the database is $rc"
+#fi
+
+# connect to the newly created database - required 
+
+echo "$DB2_HOME/bin/db2 connect to $DB2_DBNAME"
```

Comment:
changes like this in this script should be done via the CR for the db2ucluster rather than running commands against the database.

Reply:
by applying to configs like this https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/suite_db2_setup_for_manage/vars/main.yml#L3